docs/reusing_the_docker_daemon.md

```diff
@@ -31,3 +37,39 @@ The fix is to update /etc/sysconfig/docker to ensure that minikube's environment
 ```
 
 Remember to turn off the _imagePullPolicy:Always_, as otherwise Kubernetes won't use images you built locally.
+
+Another approach is to enable minikube registry addons and then push images directly into registry. Steps for this approach is as follows:
```

Comment:
Could you please make a heading so it is clear at the first look that this is the second method ?

maybe have the following headiings:

# Reusing the Docker daemon
## Method 1 : using ...

## method: 2 : using...

Reply:
Will do..